packages/x-tree-view/src/internals/models/plugin.ts

```diff
@@ -9,14 +9,33 @@ import { TreeViewItemId } from '../../models';
 import { TreeViewStore } from '../utils/TreeViewStore';
 
 export interface TreeViewPluginOptions<TSignature extends TreeViewAnyPluginSignature> {
+  /**
+   * An imperative api available for internal use.
```

Comment:
WDYT about being more explicit:
```suggestion
   * An imperative API available for internal use. Used to access methods from other plugins.
```